### ISSUE TITLE
Define image specification (technical guidance) for workflow repositories

### ISSUE THREAD

**Issue by eecavanna** [microbiomedata/docs#25]
Workflow repository maintainers are willing to produce images following some specification (e.g. dimensions, file format). I want to provide that guidance/specification to them.

**Comment by eecavanna** [comment]
@kaijli and I tested out an approach that I am satisfied and happy with, in commit https://github.com/microbiomedata/metaMAGs/commit/9fe9f0b49021a579ce8d4fda8827433991b371b7 in the `metaMAGs` repo. The approach is to have the documents reference an SVG version of the image. In that case, no dimension specification is necessary here as those images will be infinitely scalable (looking sharp at all sizes).

We also considered the fact that some people may want easy access to a rasterized (e.g. PNG) version of the image. We considered two options: (1) include a PNG alongside the SVG, although only the SVG would be referenced by the `.rst`/`.md` document; or (2) include the SVG and also include instructions someone could follow to convert the SVG into a PNG of the specific dimension they wanted. In an attempt to facilitate the transition to using SVGs in the `.rst`/`.md` files, for the time being, we will go with option (1)—making it easier for people looking for PNGs to get them. We may transition to option (2) later, especially if the diagrams start getting updated frequently.

**In summary:** When generating an image for a workflow's documentation, generate and store both an SVG and a PNG in the repository, and make the `.rst` or `.md` file reference only the SVG.
